### PR TITLE
Added a linux course

### PR DESCRIPTION
**Pull request by swarnimasaxena**
## What does this PR do?
Add resource(s)

### Description
It is a 13 chapter long tutorial and provides bonus content and a path forward as well

### Why is this valuable (or not)?
It is one of the simplest and most fun way for beginners to learn Linux

### How do we know it's really free?
It is just a text based tutorial which can be accessed by anyone through the website

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

### CODE REVIEW

**Review by Navninja7 — COMMENTED** [review]
Hello there :) If you could swap lines 826 and 827, and delete one blank line at 830, yours checks will pass.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -823,6 +823,7 @@
 * [Introduction To Linux](https://www.edx.org/course/introduction-to-linux/) - The Linux Foundation (edx)
 * [Linux Administration Tutorial Videos](https://www.youtube.com/playlist?list=PL9ooVrP1hQOH3SvcgkC4Qv2cyCebvs0Ik) - edureka!
 * [Linux Command Line](https://www.udemy.com/course/command-line/) - Adam Eubankas (Udemy)
+* [Linux Tutorial by Ryan's Tutorials](https://ryanstutorials.net/linuxtutorial)
```
Comment (shrn01) [inline courses/free-courses-en.md:826]: Please add the author's name

**Review by shrn01 — APPROVED** [review]
LGTM

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-en.md]: Don't invent titles! Use the title given by the resource. In this case, there's no obviious title, except in the html title element, so...

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -823,6 +823,7 @@
 * [Introduction To Linux](https://www.edx.org/course/introduction-to-linux/) - The Linux Foundation (edx)
 * [Linux Administration Tutorial Videos](https://www.youtube.com/playlist?list=PL9ooVrP1hQOH3SvcgkC4Qv2cyCebvs0Ik) - edureka!
 * [Linux Command Line](https://www.udemy.com/course/command-line/) - Adam Eubankas (Udemy)
+* [Linux Tutorial by Ryan's Tutorials](https://ryanstutorials.net/linuxtutorial) - Ryan Chadwick
```
Comment (eshellman) [inline courses/free-courses-en.md:826]:
```suggestion
* [Linux Tutorial for Beginners - Learn Linux and the Bash Command Line](https://ryanstutorials.net/linuxtutorial) - Ryan Chadwick
```

**Comment by Navninja7** [comment]
To get help with the webpage title, you could press F12 on the webpage whose link you have mentioned, i.e., https://ryanstutorials.net/linuxtutorial/ in our case. Then, you could select 'edit as html' from the developer options window (after pressing F12, you get the developer options, hover there and right click -> select edit as html), and look for <title> .... </title> copy the text between the tags, that is the title of the webpage.

For your webpage, this is the title:
Linux Tutorial for Beginners - Learn Linux and the Bash Command Line

Giving a title other than the one mentioned in the webpage is though sufficient to not cause any errors, but, however, it's highly recommended to use the same title used in the webpage.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books! Also thanks to @Navninja7 and @shrn01 for the assist.

**Pull request merged**